task_3_part_1.py

```diff
@@ -0,0 +1,15 @@
+soglasnye = ['b', 'c', 'd', 'f', 'g', 'h', 'k', 'l', 'm', 'n', 'p', 'r', 's', 't', 'w', 'v', 'x', 'z']
+
+
+def change_consonants_letters_to_vowel(s):
+    stroka2 = ""
+    for i in s:
+        if i in soglasnye:
+            letter = 'a'
+            stroka2 = stroka2 + letter
+        else:
+            stroka2 = stroka2 + i
+    return stroka2
```

Comment:
string have method `.replace()` you could use that :)

Reply:
Please check, I updated according your comments